SQL/Release_patches/21.0_To_22.0_upgrade.sql

```diff
@@ -35,7 +35,7 @@ SELECT 'Running: SQL/Archive/22.0/2019-07-04-remove_header_row_from_parameter_fi
 
 DELETE FROM parameter_file WHERE ParameterTypeID=(SELECT ParameterTypeID FROM parameter_type WHERE Name='header' AND SourceFrom='parameter_file');
 DELETE FROM parameter_type WHERE Name='header' AND SourceFrom='parameter_file';
-ALTER TABLE parameter_file MODIFY Value TEXT;
+
```

Comment:
"ALTER TABLE parameter_file MODIFY Value LONGTEXT;" should be here.
Just running one release patch file once，if user only wants to update Loris from 21 to 22.

Reply:
Hi @kongtiaowang , I see what you mean. Without this line, user doesn't need to apply a v24 release patch to upgrade to v22. The reason to remove this line is that v18 already has [this line handled](https://github.com/aces/Loris/blob/11e6780c0d0e2f4e33ee915f8258f9222c314892/SQL/Archive/18.0/2017-06-22_parameter_file_LONGTEXT.sql). On the other hand, do the way you suggested won't do much harm. I can change it.